<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Probes Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="valueSimulator">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Value simulator</string>
    </property>
    <widget class="QLabel" name="tempShowLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="waterLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Water Level</string>
     </property>
    </widget>
    <widget class="QLabel" name="prodShowLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="prodLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Product Level</string>
     </property>
    </widget>
    <widget class="QSlider" name="prodSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="waterSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="tempLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QLabel" name="waterShowLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="tempSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="probeConfiguration">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Probe Configuration</string>
    </property>
    <widget class="QComboBox" name="length">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lengthLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="typeMaster">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wired</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="typeSlave">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Wireless</string>
     </property>
    </widget>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QComboBox" name="id">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ErrorSimulator">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Error Simulator</string>
    </property>
    <widget class="QComboBox" name="error">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorStatusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Error Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorStatusShowLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
